src/modules/Carousel.tsx

```diff
@@ -37,12 +37,12 @@ function CarouselSlide({ slide, offset }: CarouselSlideProps) {
   const containerRef = React.useRef<HTMLDivElement>(null);
 
   const { currentIndex } = useLightboxState();
-  const { slideRect, close, focus } = useController();
+  const { slideRect, focus } = useController();
   const {
     render,
     carousel: { imageFit, imageProps },
     on: { click: onClick },
-    controller: { closeOnBackdropClick },
+    // controller: { closeOnBackdropClick },
```

Comment:
No need to keep the unused code in comments. Please delete the code that is no longer in use.